docs/search-usage.rst

```diff
@@ -0,0 +1,199 @@
+Using the API
+=============
+
+Overview
+~~~~~~~~
+
+Cloud Search allows an application to quickly perform full-text and
+geospatial searches without having to spin up instances
+and without the hassle of managing and maintaining a search service.
+
+Cloud Search provides a model for indexing documents containing structured data,
+with documents and indexes saved to a separate persistent store optimized
+for search operations.
+
+The API supports full text matching on string fields and allows indexing
+any number of documents in any number of indexes.
+
+Client
+------
+
+:class:`Client <gcloud.search.client.Client>` objects provide a means to
+configure your Cloud Search applications.  Eash instance holds both a
+``project`` and an authenticated connection to the Cloud Search service.
+
+For an overview of authentication in ``gcloud-python``, see :doc:`gcloud-auth`.
+
+Assuming your environment is set up as described in that document,
+create an instance of :class:`Client <gcloud.search.client.Client>`.
+
+.. doctest::
+
+   >>> from gcloud import search
+   >>> client = search.Client()
+
+Indexes
+~~~~~~~
+
+Indexes are searchable collections of documents.
+
+List all indexes in the client's project:
+
+.. doctest::
+
+   >>> indexes = client.list_indexes()  # API call
+   >>> for index in indexes:
+   ...     print(index.name)
+   ...     field_names = ', '.join([field.name for field in index.fields])
+   ...     print('- %s' % field_names)
+   index-name
+   - field-1, field-2
+   another-index-name
+   - field-3
+
+Create a new index:
```

Comment:
Should we make it clear that this doesn't actually "create" the index on the server? When I read "Create" I think "something will exist" and think "oh, did they forget to put `# API call`?" and then realize,... nope.